tests/test_tree_sequence.py

```diff
@@ -32,6 +32,58 @@ def test_slim_time(self, recipe):
                 mut_time = max([x['slim_time'] for x in mut.metadata['mutation_list']])
                 assert mut_time == pyslim.slim_time(ts, mut.time, stage="early")
 
+class TestNextMutationID(tests.PyslimTestCase):
+    '''
+    Tests for the function that returns the largest SLiM mutation ID.
+    '''
+    @pytest.mark.parametrize('recipe', [next(recipe_eq())], indirect=True)
```

Comment:
we want to test this on lots of recipes (like, all of them?) not just the first one (which might have no mutations!)
```suggestion
```